api/hardfork/routes.go

```diff
@@ -17,8 +19,14 @@ type TriggerHardforkHandler interface {
 }
 
 // Routes defines node related routes
-func Routes(router *gin.RouterGroup) {
-	router.POST("/trigger", Trigger)
+func Routes(router *gin.RouterGroup, routesConfig config.ApiRoutesConfig) {
+	hardforkRoutes, ok := routesConfig.APIPackages["hardfork"]
+	if !ok {
+		return
+	}
+	if configparser.CheckEndpoint("trigger", hardforkRoutes) {
```

Comment:
We could refactor this a bit to look cleaner. Instead of having `if active -> then` on every route: We can have a general wrappers for `Get` and `Post` that make this check and have the condition only in one place

Reply:
done. made a wrapper over RouterGroup